recipes/client.rb

```diff
@@ -20,6 +20,7 @@
 
 if node['osl-mysql']['enable_percona_client']
   include_recipe 'percona::client'
+  package "Percona-Server-devel-#{node['percona']['version'].tr('.', '')}"
```

Comment:
Why is this being added?

Reply:
Previous behavior would allow the `percona-server-devel-56` package to be added to the list of packages for the `percona::client` recipe to install, however because of the 1.0 changes if there are any packages listed in `['percona']['client']['packages']` then it doesn't install `percona-server-client-56`. I could also add `percona-server-client-56` to the list of packages to install in the default recipe as an alternate solution

Reply:
Maybe it would be a good idea to create a devel recipe or an attribute which enables installing it upstream? I'd rather do that than do this here.